cmd/create/cluster/cmd.go

```diff
@@ -2080,6 +2090,30 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	httpTokens := args.httpTokens
```

Comment:
Should we add !hosted-cp check to ensure it is not displayed for hypershift cluster

Reply:
@tsorya we can also validate at the beginning of the function that the flags `hosted-cp` and `http-tokens` are not provided together, to return an error to the user in an early stage.

Reply:
Done